core/templates/services/assets-backend-api.service.spec.ts

```diff
@@ -194,8 +194,10 @@ describe('Assets Backend API Service', function() {
 
     it('should successfully save an audio', function(done) {
       var successMessage = 'Audio was successfully saved.';
-      // @ts-ignore in order to ignore JQuery properties that should
-      // be declarated.
+      // There is an error here because the type of function we are using here
+      // doesn't match the type of actually $.ajax. We need to do this for
```

Comment:
... doesn't match the type of the actual $.ajax. ...

Also it's kind of vague. What do you mean by "the type of function we are using here"? Give more detail.

Reply:
done